edx-solutions/xblock-drag-and-drop-v2
# Support for multiple DnD blocks on the same page

**Description:** this PR fixes a bug with having multiple DnD blocks on the same page - items from second and subsequent blocks, when dragged, jumped into the first block drop area.

**JIRA:** [SOL-1148](https://openedx.atlassian.net/browse/SOL-1448)

**Testing instructions:**
1. Add two or more DnD blocks in the unit.
2. Publish, open the unit in LMS.
3. Try dragging items in the first block - should work just fine (including correcr/incorrect zones and final feedback) - not changed.
4. Try dragging items in the second block. *Expected behavior:* works just fine. *Pre-fix behavior:* items jump to first block drop area; impossible to complete the second block.
